lib/storage/metadata/file/MetadataFileServer.js

```diff
@@ -98,16 +106,31 @@ class MetadataFileServer {
         fs.accessSync(this.path, fs.F_OK | fs.R_OK | fs.W_OK);
         storageUtils.setDirSyncFlag(this.path, this.logger);
 
+        this.servers = [];
```

Comment:
I would suggest moving these to the constructor. It will give a clearer idea of the object's skeleton.